src/pages/find-offers/FindOffers.tsx

```diff
@@ -72,18 +70,26 @@ const FindOffers = () => {
     })
 
   const getOffers = useCallback(
-    (params?: GetOffersParams) => OfferService.getOffers(params),
-    []
+    () =>
+      OfferService.getOffers({
+        ...filters,
+        status: StatusEnum.Active,
+        limit: itemsPerPage,
+        skip: (Number(filters.page) - 1) * itemsPerPage
+      }),
+    [filters]
   )
 
   const {
-    response: offersResponse,
-    loading: offersLoading,
-    fetchData
-  } = useAxios<GetOffersResponse, GetOffersParams>({
-    service: getOffers,
-    defaultResponse,
-    fetchOnMount: false
+    isLoading: offersLoading,
+    data: offersResponse,
+    refetch: fetchData
+  } = useQuery({
+    queryKey: ['offers', filters, searchParams.toString()],
```

Comment:
Is it working without `toString()`? Probably `useQuery` will auto-serialize keys, so you don't need to do it by yourself.
```suggestion
    queryKey: ['offers', filters, searchParams],
```

Reply:
Also, look at your keys, they contain `offers` as it was in the previous component. Don't change it in this component, but change it in the previous one, please

Reply:
Yes, it works without toString(). I'm going to fix that.

Reply:
Actually, I'm not sure whether I should pass searchParams to queryKey, what do you think?